.devcontainer/installMoreTools.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+mkdir install-more-tools
+cd install-more-tools
+
+SCORE_COMPOSE_VERSION=$(curl -sL https://api.github.com/repos/score-spec/score-compose/releases/latest | jq -r .tag_name)
+wget https://github.com/score-spec/score-compose/releases/download/${SCORE_COMPOSE_VERSION}/score-compose_${SCORE_COMPOSE_VERSION}_linux_amd64.tar.gz
+tar -xvf score-compose_${SCORE_COMPOSE_VERSION}_linux_amd64.tar.gz
+sudo chmod +x score-compose
```

Comment:
do we really need sudo here? we shouldn't...

Reply:
we should be running as a user that can access /usr/local just fine

Reply:
For the `chmod`s you are right, but for the `mv`, I just tested and we still need `sudo` apparently, am I missing something?

Reply:
I just merged it, but if we see we could improve this `sudo mv` parts, we could open a new PR later. Thanks for your review!